package/libpiksi/libpiksi/src/endpoint.c

```diff
@@ -125,6 +129,8 @@ struct pk_endpoint_s {
   void *metrics_timer;     /**< Handle of the metrics flush timer */
   bool warned_on_discard;  /**< Warn only once for writes on read-only sockets */
   char identity[PATH_MAX]; /**< The 'identity' of this socket, used for recording metrics */
+  pk_endpoint_eagain_fn_t
+    eagain_cb; /**< Function that's called when we terminate a connection on an EAGAIN error */
```

Comment:
Maybe `Callback for terminating a connection on an EAGAIN error` to make it fit one line

Reply:
🔨